src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj

```diff
@@ -16,5 +16,5 @@
   
   <Import Condition="'$(_packageTargetOSGroup)' != ''" Project="$(MSBuildThisFileDirectory)runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props" />
 
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\dir.targets" />
```

Comment:
we don't want `..\..\dir.targets`, we want `..\..\..\dir.targets`.  This was part of the reason I had named `..\..\dir.targets` as `..\..\packaging.targets`.